mkdocs.yml

```diff
@@ -56,20 +72,20 @@ nav:
       - Longitudinal and multi-site studies: 06-longitudinal-and-multi-site-studies.md
       - BIDS Extension Proposals: 07-extensions.md
       - Appendix:
-        - Contributors: 99-appendices/01-contributors.md
-        - Licenses: 99-appendices/02-licenses.md
-        - Hierarchical Event Descriptors: 99-appendices/03-hed.md
-        - Entity table: 99-appendices/04-entity-table.md
-        - Units: 99-appendices/05-units.md
-        - MEG file formats: 99-appendices/06-meg-file-formats.md
-        - MEG systems: 99-appendices/07-meg-systems.md
-        - Coordinate systems: 99-appendices/08-coordinate-systems.md
-        - Entities: 99-appendices/09-entities.md
-        - File collections: 99-appendices/10-file-collections.md
-        - Quantitative MRI: 99-appendices/11-qmri.md
-        - Arterial Spin Labeling: 99-appendices/12-arterial-spin-labeling.md
-        - Cross modality correspondence: 99-appendices/13-cross-modality-correspondence.md
-        - Glossary: 99-appendices/14-glossary.md
+        - Contributors: appendices/contributors.md
+        - Licenses: appendices/licenses.md
+        - Entity table: appendices/entity-table.md
+        - Entities: appendices/entities.md
+        - File collections: appendices/file-collections.md
+        - Units: appendices/units.md
+        - Hierarchical Event Descriptors: appendices/hed.md
+        - MEG file formats: appendices/meg-file-formats.md
+        - MEG systems: appendices/meg-systems.md
+        - Coordinate systems: appendices/coordinate-systems.md
+        - Quantitative MRI: appendices/qmri.md
+        - Arterial Spin Labeling: appendices/arterial-spin-labeling.md
+        - Cross modality correspondence: appendices/cross-modality-correspondence.md
+      - Glossary: glossary.md
```

Comment:
I would even bump the glossary before the BEP.

Reply:
I'll do that then. Thanks!